nipoppy/cli.py

```diff
@@ -343,3 +344,54 @@ def status(**params):
     params = dep_params(**params)
     with handle_exception(StatusWorkflow(**params)) as workflow:
         workflow.run()
+
+
+@cli.group(cls=OrderedGroup)
+def pipeline():
+    """Local pipeline management."""
+    pass
+
+
+def zenodo_options(func):
+    """Define global options for the CLI."""
+    func = click.option(
+        "--zenodo-token",
+        envvar="ZENODO_TOKEN",
+        type=str,
+        required=False,
+        help="Zenodo access token.",
+    )(func)
+    func = click.option(
+        "--zenodo-api",
+        envvar="ZENODO_API",
+        default="https://sandbox.zenodo.org/api",
+        type=str,
+        required=False,
+        help="Zenodo access token.",
+    )(func)
+    return func
+
+
+@pipeline.command("add")
+@click.argument(
+    "zenodo_id",
+    type=str,
+)
+@dataset_option
+@zenodo_options
+def pipeline_add(**params):
+    """Add a new pipeline to the configuration."""
```

Comment:
```suggestion
    """Add a new pipeline."""
```
Since we are not updating the global config (yet, to discuss)